.env.zappdev.example

```diff
@@ -0,0 +1,21 @@
+# Remaining env keys below can remain as they are
```

Comment:
Moreover as soon as we integrate the new `config` module (the task is almost solved ATM) all the configuration properties will be stored in dedicated files, one for each environment. So in docker-compose files we will only need to specify the environment name, and the app will pick the corresponding configuration automatically.

Reply:
One advantage of env variable is that could be loaded directly to memory without changing the application. For example, when deploying apps to cloud environments there are specific features to mange those variables. In other app that I worked before, it was possible to move from dev to prod environment just changing env variables. If this kind of use case will not occur and the config files has fixed values that will not change, no problem with that. In case, I'll maintain this file for now until the new config module be available. If the config feature be merged before this PR, I'll pull and follow the new configuration standard.

Reply:
@diogosmendonca you can still use whatever env variable you need. Currently we've integrated a new configuration approach which allows to override any configuration properties you need. Please see [this package description](https://www.npmjs.com/package/config) and also check out configurations we currently use [here](https://github.com/pointnetwork/pointnetwork/tree/develop/config). Please let me know if you cannot achieve what you need using the described approach.

Reply:
@tankakatan Thaks for the answer. I created one env file for zappdev env based on the production one. This was discussed with @jensendarren. The motivation was to set other tags for the point network image and rebuild them as part of the development process.